app/renderer/src/assets/locales/en/translation.json

```diff
@@ -282,5 +282,13 @@
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
   "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
-  "duplicateCapabilityNameError": "A capability set with the same name already exists"
+  "duplicateCapabilityNameError": "A capability set with the same name already exists",
+  "errorUploadingGesture": "Error uploading gestures",
+  "unableToUploadGestureFiles": "Could not upload gesture from below files:",
```

Comment:
the list of files should also be in the string template

Reply:
The message acts more like a label followed by collapsible HTML tags with file details.